unitTests/ui/CategoryFacet/CategoryFacetTest.ts

```diff
@@ -613,5 +618,28 @@ export function CategoryFacetTest() {
         expect(ellipsis.nextSibling.textContent).toContain('parent25');
       });
     });
+
+    describe('testing the DependsOnManager', () => {
+      beforeEach(() => {
+        spyOn(test.cmp.dependsOnManager, 'updateVisibilityBasedOnDependsOn');
+        spyOn(test.cmp.dependsOnManager, 'listenToParentIfDependentFacet');
+      });
+
+      it('should initialize the dependsOnManager', () => {
+        expect(test.cmp.dependsOnManager).toBeTruthy();
```

Comment:
Is it cleaner to use the shorter but less specific `.toBeTruthy()` or more specific but longer `.not.toBeNull()`?

Reply:
`. toBeTruthy()` is "good enough" in this case because we want to check if it is defined. Here's what truthy does, it's the same `!!` we discussed in a previous PR https://github.com/jasmine/jasmine/blob/4097718b6682f643833f5435b63e4f590f22919f/lib/jasmine-core/jasmine.js#L2914

In this the dependsOnManager is either not defined (`undefined`) of defined which truthy checks correctly. A specific `null` wouldn't pass as it is different then `undefined` in JS.
We could be more specific and use `not.toBeUndefined` or check that the object it creates has all the right params, but like I said `.toBeTruthy()`  does the job well in this case.